xbmc/epg/EpgInfoTag.cpp

```diff
@@ -749,3 +756,13 @@ void CEpgInfoTag::SetEpg(CEpg *epg)
 {
   m_epg = epg;
 }
+
+bool CEpgInfoTag::IsSeries(void) const
+{
+  return m_iFlags & EPG_TAG_FLAG_IS_SERIES > 0;
```

Comment:
Although actually not necessary, could you please put brackets around <code>m_iFlags & EPG_TAG_FLAG_IS_SERIES</code>. Increases readability, imo.